app/uk/gov/hmrc/nisp/config/ApplicationConfig.scala

```diff
@@ -45,6 +47,10 @@ trait ApplicationConfig {
   val ivUpliftUrl: String
   val twoFactorUrl: String
   val ggSignInUrl: String
+  val breadcrumbServiceUrl: String
```

Comment:
For now breadcrumbService and pertaxFrontend are the same hosts, so could we combine these two config properties?